data/frame_meta.go

```diff
@@ -11,11 +11,15 @@ import (
 // this interface only exposes the values we want to be exposed
 type FrameMeta struct {
 	// Datasource specific values
-	Custom map[string]interface{} `json:"custom,omitempty"`
+	Custom interface{} `json:"custom,omitempty"`
```

Comment:
I thought we changed this before... the values is parsed into an javascript object, so it can not be an array or single value.  However forcing it to be a map is not great either.

Reply:
`map[string]interface{}` is the data type to use for expressing JSON document/properties in Go.

Reply:
I'm fine either way.  We have two options:

1. `interface{}`
pro: easy to work with, it is run though json serialization directly
con: *can* put in invalid data if the value is not an interface

2. `map[string]interface{}`
pro: forces an object
con: can not use a well typed object in go to specify known properties

Given the trade offs, I think option 1 is better (but can happily go either way)

Reply:
I'm not sure what you're suggesting to change. Maybe @kylebrandt have some context to how this have been implemented and why?

Reply:
Can we remove this change from this PR to get this merged? 

I agree with Ryan that I think an interface{} would be better for the reasons stated, but would need to think through how that will play out as a change. But better as different issue or pr.